server/services/rabbitmq_service.go

```
@@ -0,0 +1,179 @@
+
+	var forever chan struct{}
+
+	go func() {
+		for d := range msgs {
+			err := args.Callback(string(d.Body))
+			if err != nil {
+				log.Printf("[*][%s] Error %s", args.Queue, err)
+				break
+			}
+
+			d.Ack(false)
```

golangci-lint (1.62.2)

Error return value of `d.Ack` is not checked

(errcheck)